cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -225,24 +227,25 @@ protected void addService(String name, HttpServiceHandler handler, HttpServiceHa
    * @param schedule the schedule to be added for the Workflow
    * @param workflowName the name of the Workflow
    */
+  @Deprecated
```

Comment:
``TimeSchedule`` and ``StreamSizeSchedule`` already extend ``Schedule``, yes. The reason I thought we wanted to deprecate the method is to not expose ``Schedule`` directly to users, so that they can't define their own schedule by extending the class.